src/tags/forms.py

```diff
@@ -14,3 +19,23 @@ class Media:
         css = {
             'all': ('https://cdnjs.cloudflare.com/ajax/libs/select2-bootstrap-theme/0.1.0-beta.10/select2-bootstrap.min.css',)
         }
+
+
+class TagForm(forms.ModelForm):
+
+    class Meta:
+        model = Tag
+        fields = '__all__'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields['slug'].required = False
```

Comment:
What's the purpose of adding the slug field? Can we remove it? I don't think the user needs to know about it at all, if it's required for the form, can we just hide it?

Reply:
its set as unique=True and non optional (at least in the github version)
![image](https://user-images.githubusercontent.com/39788517/182727329-6a5d5fa1-ddb2-4831-986d-85c646a4a587.png)
ill remove it from the form, but set it in the save func
![image](https://user-images.githubusercontent.com/39788517/182727295-51b88bad-6052-4d43-8a48-2fe14bd404c1.png)